flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/delegation/ExecutorBase.java

```diff
@@ -37,38 +40,56 @@
 
 	private static final String DEFAULT_JOB_NAME = "Flink Exec Table Job";
 
-	private final StreamExecutionEnvironment executionEnvironment;
-	protected List<Transformation<?>> transformations = new ArrayList<>();
+	private final StreamExecutionEnvironment execEnv;
```

Comment:
> 	private final StreamExecutionEnvironment execEnv;

Maybe the variable name can keep original name which has same style with other sub classes of Executor.